src/screens/StudentGroups.tsx

```diff
@@ -8,72 +8,61 @@ import { FormErrorMessage, FormLabel, FormControl, Input, Button } from '@chakra
 
 import StudentGroup from 'components/StudentGroup'
 
-const Classrooms: React.FC = () => {
-  const { handleSubmit, errors, register, formState, reset } = useForm<FormData>()
+const StudentGroups: React.FC = () => {
+  const { handleSubmit, errors, register, formState } = useForm<FormData>()
 
-  const user = useUser()
+  const { data: user } = useUser()
+
+  const studentGroupsRef = useFirestore().collection('teachers').doc(user.uid).collection('studentGroups')
+  const studentGroupsData = useFirestoreCollectionData(studentGroupsRef)
+  console.log(studentGroupsData)
 
   type FormData = {
     studentGroupName: string
   }
-  interface IClassroom {
-    docId: string
+
+  //Steve is the druel-ist.
+  interface IStudentGroup {
     studentGroupName: string
     students: []
+    docId: string
```

Comment:
I think it's better to append this field where the data itself is appened on line 32.  I've done if for us already with the `&` union operator.

Reply:
Is that because it better reflects the fact that the data is being added on by firebase?  I'm not sure I understand this all the way

Reply:
Yes, exactly

Reply:
Awesome, got it